pywren/wrenhandler.py

```diff
@@ -244,33 +244,33 @@ def generic_handler(event, context_dict):
         logger.info("data data download complete, took {:3.2f} sec".format(data_download_time))
         response_status['data_download_time'] = data_download_time
 
-        # now split
-        d = json.load(open(func_filename, 'r'))
+        # d = json.load(open(func_filename, 'r'))
```

Comment:
So I'm not sure I fully follow why we still have the data_key download and func_key download happening in wrenhandler -- isn't the func object the thing we want to move to memory as it gets to 25MB or so from notebooks ?

Reply:
Sorry, again, this is in-progress, there's some detritus that needs to be cleaned out. I should have been more clear, i'm sorry!